src/app/rundown-view/components/rundown-tooltips/mini-shelf-tooltip/mini-shelf-tooltip.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input } from '@angular/core'
+import { Subject } from 'rxjs'
+import { Tv2PieceType } from 'src/app/core/enums/tv2-piece-type'
+
+@Component({
+  selector: 'sofie-mini-shelf-tooltip',
+  templateUrl: './mini-shelf-tooltip.component.html',
+  styleUrls: ['./mini-shelf-tooltip.component.scss'],
+})
+export class MiniShelfTooltipComponent {
+  @Input() public fileName: string
+
+  public readonly type: Tv2PieceType = Tv2PieceType.VIDEO_CLIP
+  public mouseHoverEventSubject: Subject<MouseEvent | undefined> = new Subject<MouseEvent | undefined>()
```

Comment:
Minor thing: When declaring a subject like this you can omit the types on the instantiation because they are already determined on the left side of the equal sign:
```
public mouseHoverEventSubject: Subject<MouseEvent | undefined> = new Subject()
```